pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/hardware.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+import { LineGraph } from "src/views/cluster/components/linegraph";
+import { Metric, Axis, AxisUnits } from "src/views/shared/components/metricQuery";
+
+import { GraphDashboardProps, nodeDisplayName } from "./dashboardUtils";
+
+// TODO(vilterp): tooltips
+
+export default function (props: GraphDashboardProps) {
+  const { nodeIDs, nodesSummary, nodeSources, tooltipSelection } = props;
+
+  return [
+    <LineGraph
+      title="CPU Percent"
+      sources={nodeSources}
+    >
+      {/* TODO(vilterp): multiply by 100 to get percent */}
+      <Axis units={AxisUnits.Count} label="CPU">
```

Comment:
Oh, hey, at one point I removed an AxisUnits.Percent cause it wasn't used anywhere...  Here ya go: https://github.com/cockroachdb/cockroach/commit/70c44194d917ef9f9463c9d18b8ed29079f5faaa

Reply:
Done